app/views/group_assignment_invitations/_group_assignment_team.erb

```diff
@@ -0,0 +1,31 @@
+<% group_assignment_repo = repo.decorate %>
+
+<div class="one-half left team">
+  <div class="team-panel">
+    <% group_member_count = group_assignment_repo.group.repo_accesses.count %>
+    <%= form_tag({ controller: :group_assignment_invitations, action: :accept_invitation}, :class => "team-actions-form", method: :patch) do %>
+        <%= hidden_field_tag 'group[id]', group_assignment_repo.id %>
+        <% if group_member_count >= group_assignment.max_members %>
+            <%= submit_tag 'Full', class: 'btn', disabled: true %>
+        <% else %>
+            <%= submit_tag 'Join', class: 'btn' %>
+        <% end %>
+    <% end %>
+
+    <span class="team-name" itemprop="name"><%= group_assignment_repo.team_name %></span>
+    <p class="team-description" itemprop="description"><%=group_member_count%> <% if group_member_count != 1%>members<% else %>member<% end %></p>
+    <div class="team-members">
+      <% group_assignment_repo.group.repo_accesses.limit(7).each do |repo_access| %>
+          <% student = repo_access.user.decorate %>
+          <%= link_to student.github_url, class: "tooltipped tooltipped-s #{'disabled' if group_assignment_repo.disabled?}", 'aria-label' => student.login do %>
+              <%= image_tag student.avatar_url(30), class: 'avatar avatar-small ', height: 30, width: 30, alt: "@#{student.login}" %>
+          <% end %>
+      <% end %>
+      <% if group_member_count > 10 %>
+        <%= link_to group_assignment_repo.github_team_url, class: "#{'disabled' if group_assignment_repo.disabled?}" do %>
+          <%= image_tag 'student-ellipse@2x.png', class: 'avatar avatar-small', height: 30, width: 30, alt: 'student ellipse' %>
+        <% end %>
+      <% end %>
+    </div>
+  </div>
+</div>
```

Comment:
Can we make the build fail on missing newline @tarebyte?

cc/ https://github.com/education/classroom/issues/514